Source/WebKit/UIProcess/API/C/WKContext.cpp

```diff
@@ -457,16 +457,6 @@ void WKContextAllowSpecificHTTPSCertificateForHost(WKContextRef, WKCertificateIn
 {
 }
 
-void WKContextDisableProcessTermination(WKContextRef contextRef)
-{
-    WebKit::toImpl(contextRef)->disableProcessTermination();
-}
-
-void WKContextEnableProcessTermination(WKContextRef contextRef)
```

Comment:
How did you confirm that all these changes to our C API were OK?
Do we know for sure there are no longer any (Apple-internal) projects relying on these?

Reply:
I am not sure to be honest.